frontend/src/services/utils.ts

```diff
@@ -50,3 +52,8 @@ export const formValidationSchema = (isAbstractRoleSelected: boolean) => {
 export const isNewAccess = (newAccessIds: string[], accessId: number) => {
     return newAccessIds.includes(accessId.toString());
 };
+
+export const isSelectedAppProd_OnProdEnvironment = () => {
```

Comment:
Sorry for this odd looking variable name. I couldn't come up with a short and sweet name... normally I use camelCase, but I separate it with underscore to be a bit clear... if you come up with a better name please let me know...

Reply:
isProdAppSelectedOnProdEnv?

Reply:
Sure, I can change to that